tests/resources/serializers/test_datacite_serializer.py

```diff
@@ -257,3 +261,88 @@ def test_datacite43_serializer(running_app, full_record):
+        "<?xml version='1.0' encoding='utf-8'?>",
+        "<resource xmlns=\"http://datacite.org/schema/kernel-4\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.3/metadata.xsd\">",  # noqa
+        "  <identifier identifierType=\"DOI\">10.5281/inveniordm.1234</identifier>",  # noqa
+        "  <alternateIdentifiers>",
+        "    <alternateIdentifier alternateIdentifierType=\"bibcode\">1924MNRAS..84..308E</alternateIdentifier>",  # noqa
+        "  </alternateIdentifiers>",
+        "  <creators>",
+        "    <creator>",
+        "      <creatorName nameType=\"Personal\">Nielsen, Lars Holm</creatorName>",  # noqa
+        "      <givenName>Lars Holm</givenName>",
+        "      <familyName>Nielsen</familyName>",
+        "      <nameIdentifier nameIdentifierScheme=\"ORCID\">http://orcid.org/0000-0001-8135-3489</nameIdentifier>",  # noqa
+        "    </creator>",
+        "  </creators>",
+        "  <titles>",
+        "    <title>InvenioRDM</title>",
+        "    <title xml:lang=\"eng\" titleType=\"Subtitle\">a research data management platform</title>",  # noqa
+        "  </titles>",
+        "  <publisher>InvenioRDM</publisher>",
+        "  <publicationYear>2018</publicationYear>",
+        "  <subjects>",
+        "    <subject>custom</subject>",
+        "    <subject subjectScheme=\"MeSH\">Abdominal Injuries</subject>",
+        "  </subjects>",
+        "  <contributors>",
+        "    <contributor contributorType=\"Other\">",
+        "      <contributorName nameType=\"Personal\">Nielsen, Lars Holm</contributorName>",  # noqa
+        "      <givenName>Lars Holm</givenName>",
+        "      <familyName>Nielsen</familyName>",
+        "      <nameIdentifier nameIdentifierScheme=\"ORCID\">http://orcid.org/0000-0001-8135-3489</nameIdentifier>",  # noqa
+        "    </contributor>",
+        "  </contributors>",
+        "  <dates>",
+        "    <date dateType=\"Issued\">2018/2020-09</date>",
+        "    <date dateType=\"Other\" dateInformation=\"A date\">1939/1945</date>",  # noqa
+        "  </dates>",
+        "  <language>dan</language>",
+        "  <resourceType resourceTypeGeneral=\"Image\">Photo</resourceType>",
+        "  <relatedIdentifiers>",
+        "    <relatedIdentifier relatedIdentifierType=\"DOI\" relationType=\"Cites\" resourceTypeGeneral=\"Dataset\">10.1234/foo.bar</relatedIdentifier>",  # noqa
+        "  </relatedIdentifiers>",
+        "  <sizes>",
+        "    <size>11 pages</size>",
+        "  </sizes>",
+        "  <formats>",
+        "    <format>application/pdf</format>",
+        "  </formats>",
+        "  <version>v1.0</version>",
+        "  <rightsList>",
+        "    <rights rightsURI=\"https://creativecommons.org/licenses/by/4.0/\" rightsIdentifierScheme=\"spdx\" rightsIdentifier=\"cc-by-4.0\">Creative Commons Attribution 4.0 International</rights>",  # noqa
```

Comment:
The tests are failing here however. I saw another PR that made a change the record ``rights`` property, thus that's likely the cause. However, the string above is correct, and we should serialize as above, so we'll need to check how to fix it.

Reply:
It was modified to make tests passed. @jrcastro2 in your PR they should pass with 
```
 "  <rightsList>",
"    <rights rightsURI=\"https://creativecommons.org/licenses/by/4.0/\" rightsIdentifierScheme=\"spdx\" rightsIdentifier=\"cc-by-4.0\">Creative Commons Attribution 4.0 International</rights>",  # noqa
"  </rightsList>",
```
You might need to add something to the conftests values maybe? Worth to add a custom one too?